polygon/bor/bor.go

```diff
@@ -1048,10 +1048,6 @@ func (c *Bor) Finalize(config *chain.Config, header *types.Header, state *state.
 		return nil, types.Receipts{}, nil, err
 	}
 
-	// No block rewards in PoA, so the state remains as is and uncles are dropped
-	// header.Root = state.IntermediateRoot(chain.Config().IsSpuriousDragon(header.Number.Uint64()))
-	header.UncleHash = types.CalcUncleHash(nil)
```

Comment:
Let's remove the same line from `(c *Bor) FinalizeAndAssemble`

Reply:
@yperbasis i thought need remove it from Finalize and leave in FinalizeAndAssemble. Am i wrong?

Reply:
`NewBlock` sets `UncleHash` anyway, so there's no need to set it explicitly in `FinalizeAndAssemble`

Reply:
ah. removed.